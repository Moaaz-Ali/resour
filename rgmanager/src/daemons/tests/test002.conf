<?xml version="1.0"?>
<!-- normal resource group -->
<!-- If compared with the inline resource group in test1,
     this should not make this resource group restart since no
     attributes have changed. -->
<cluster>
<rm>
	<resources>
    		<resourcegroup name="test1"/>
	</resources>
	<resourcegroup ref="test1"/>
</rm>
</cluster>
