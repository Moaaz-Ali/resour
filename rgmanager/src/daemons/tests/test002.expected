=== Resources List ===
Resource type: resourcegroup [ROOT]
Instances: 1/1
Agent: resourcegroup.sh
Attributes:
  name = test1 [ primary unique required ]

=== Resource Tree ===
resourcegroup {
  name = "test1";
}
